canvasapi/canvas.py

```diff
@@ -498,7 +545,7 @@ def get_appointment_group(self, appointment_group, **kwargs):
         )
         return AppointmentGroup(self.__requester, response.json())
 
-    def get_appointment_groups(self, **kwargs):
+    def get_appointment_groups(self, **kwargs: Optional[dict]) -> PaginatedList:
```

Comment:
Is it possible to annotate with `PaginatedList[AppointmentGroup]`

Reply:
Yes, you can do that directly or through a [Type Alias](https://docs.python.org/3/library/typing.html#type-aliases) to simplify complex returns.